### ISSUE TITLE
ldapadmin - order a new account : pb with mail adress format or domain name

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/georchestra#1520]
Wanted to create a new account for testing. Were rejected overthere thsi mail adress is correct

![screenshot280](https://cloud.githubusercontent.com/assets/6370443/19432750/3136365a-945e-11e6-877e-86a15177d3d8.png)

The .bzh is not allowed. Why ?

**Comment by fvanderbiest** [comment]
because https://github.com/georchestra/georchestra/blob/15.12/ldapadmin/src/main/webapp/WEB-INF/views/validation.jsp#L269

**Comment by fvanderbiest** [comment]
See also https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains

**Comment by MaelREBOUX** [comment]
may I PR ?

**Comment by fvanderbiest** [comment]
Considering the amount of TLDs to handle, a convenient fix would be to disable TLD checks IMO.

**Comment by landryb** [comment]
Agreed, this doesnt make sense to me to have a small whitelist... that needs constant fixing.

**Comment by fvanderbiest** [comment]
Still ready for a PR @MaelREBOUX ?

**Comment by fvanderbiest** [comment]
Well, by default `checkTLD` is false, which results in `emailCheck('mael@breizhpositive.bzh') = true`.

You probably have a space character at the beginning of your email address.
 `emailCheck(' mael@breizhpositive.bzh') = false`.

If this deserves a PR, then it's to trim the provided value.

**Comment by lecault** [comment]
Same problem in GeoBretagne. We could modify email adress but not create new user with bzh adress (and not change password).

Agree to disable TLD checked.

**Comment by fvanderbiest** [comment]
There's no TLD check to my knowledge ...

**Comment by lecault** [comment]
PR minimale envoyée :)

**Comment by MaelREBOUX** [comment]
I reopened because it seems not fonctionnal.
Anyone could test and report here please ?

I have a doubt on my geor site if it is up-to-date with this merge.

**Comment by fvanderbiest** [comment]
> I reopened because it seems not fonctionnal.

Indeed it does not work. Dunno why.

**Comment by MaelREBOUX** [comment]
poke @fpelhate

**Comment by Vampouille** [comment]
Should be fixed by : https://github.com/georchestra/georchestra/pull/1738

**Comment by MaelREBOUX** [comment]
On which branches has it been put ?
I read 15.12 on the commit ? Target is not master or 16.12 ?

**Comment by fvanderbiest** [comment]
> Should be fixed by : #1738

This fix is not enough, angular should also be upgraded to >= 1.2.0 see https://code.angularjs.org/1.2.0/docs/api/ng.directive:input.email

There are risks of regressions...

**Comment by fvanderbiest** [comment]
GeoGrandEst is also affected with `.alsace` tld

**Comment by MaelREBOUX** [comment]
Thanks @lecault
